### DISCUSSION TITLE
Alerting  message description

### DISCUSSION THREAD

**Discussion by wangan0915** [apache/skywalking#9444] [Q&A]
rules:
  # Rule unique name, must be ended with `_rule`.
  service_resp_time_rule:
    metrics-name: service_resp_time
    op: ">"
    threshold: 1000
    period: 10
    count: 3
    silence-period: 5
    message: Response time of service {name} is more than 1000ms in 3 minutes of last 10 minutes.
question: look at message content, i dont understant why using "3 minutes" instead of  "3 times".

**Comment by wu-sheng** [comment]
Read the latest doc, https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-alarm/#individual-rules.
It should be able to help you to understand the alerting mechanism.

**Reply by wangan0915** [reply]
all Alarm config are default,bug in a period,i got many alarm data,for example:
Message:Successful rate of service prod_inco::fast is lower than 80% in 2 minutes of last 10 minutes
Event Detail:
Event IDEvent NameEvent TypeStart TimeEnd Time
30ef45f9-d02f-4411-a173-6bcea579198aAlarmError2022-10-08 23:48:17  2022-10-08 23:58:17
37fb6c9a-adbe-4b70-b827-da5e3d32ee07AlarmError2022-10-08 23:47:25  2022-10-08 23:57:25
i don't know why the alarm not only is triggered at 23:48:17,but alse is triggered at 23:47:25,
period is 10.

**Reply by wu-sheng** [reply]
That relies on the flush period too. The slots of the alarting engine get updated after flushing into the storage.

**Reply by wangan0915** [reply]
i don't understand about alarting engine and flush period,but That's what you mean,i will received two alarm message in two minutes even if the period  is 10 and the silence-period is 3，This is normal？

**Reply by wu-sheng** [reply]
You need to find out why, such as, whether these notifications are sent from two OAP(s)? This would be common if you don't set the cluster mode currently.
We can't and shouldn't debug for you. If you want help from the community, try to locate the issue first.
`don't say it is not working`, `it is un-normal`. I would not be surprised if SkyWalking has bugs, also, there is no help and fix if you can't locate something in your scenario.
